### PR TITLE
Improvements to Finnish voice guidance

### PR DESCRIPTION
**Pull request by akikoskinen**
Check commit message for full details.

I transferred the file to a device and used the development tools in OsmAnd to listen through all the examples in the voice prompts section.

### CODE REVIEW

File: voice/fi/ttsconfig.p [hunk 1]
```diff
@@ -86,27 +86,26 @@ string('follow.ogg', 'Seuraa tietä ').
 
 % ARRIVE
 string('and_arrive_destination.ogg', 'ja olet perillä ').
-string('reached_destination.ogg', 'olet perillä ').
+string('reached_destination.ogg', 'olet perillä, ').
```
Comment (xmd5a2) [inline voice/fi/ttsconfig.p]: Is this change really necessary?

Comment (akikoskinen) [reply voice/fi/ttsconfig.p]: It's for adding a little pause before the destination name which can come after this. Without the comma it's just continuous: "olet perillä koti" ("koti" means home). With the comma, there's a slight pause: "olet perillä, koti". If there's another way to add that pause, then I'm willing to learn that.

Comment (akikoskinen) [reply voice/fi/ttsconfig.p]: And the whole phrase just sounds more humane with that pause. Without the pause it sounds like a machine -- which of course it is :)
